laurent22/joplin
# Desktop: Fixes #10895: Fix alt+up/alt+down fails to re-order multiple lines

# Summary

This pull request fixes #10895 by replacing the custom `swapLine` editor command with the built-in CodeMirror [`moveLineUp`](https://codemirror.net/docs/ref/#commands.moveLineUp) and `moveLineDown` commands.

# Testing plan

1. Start the desktop app.
2. Select multiple lines.
3. Press <kbd>alt</kbd>-<kbd>up</kbd>. Verify that all selected lines are moved up.
4. Press <kbd>alt</kbd>-<kbd>down</kbd>. Verify that all selected lines are moved down.
